apps/docs/src/guide/cli/commands.md

```diff
@@ -56,7 +56,8 @@ npx fuels help build
 
 ```console
 Options:
-  -p, --path <path>  Path to project root (default: "/Users/anderson/Code/fuel/fuels-ts/apps/docs")
+  -p, --path <path>  Path to project root (default: "/Users/anderson/Code/fuel/fuels-ts")
```

Comment:
```suggestion
  -p, --path <path>  Path to project root (default: "/Users/user/Code/fuel/fuels-ts")
```

Nit, make more general

Reply:
Oh, good catch! 🙌

This was wrong from the very beginning!

https://github.com/FuelLabs/fuels-ts/pull/1784/commits/32639be7647bf5d4c4177abd24a53bd9c0044dad